### ISSUE TITLE
Final Project

### ISSUE THREAD

**Issue by JayCastro** [DS4PS/cpp-526-sum-2020#24]
Quick question. The two tabs we make can be any combination of things right?

the instruction say 
"Add at least two additional tabs that allow for meaningful analysis of the data. Include widgets for user input, and link the widgets to a leaflet map (like tabs 1 and 2) or any other tables or graphs that you find useful for analyzing traffic patterns. You might consider:

Accident characteristics:

Type (Collisionmanner)
Violations issued
Conditions at the time of accident:

Time of day
Weather
Light
Surface
Junction
Impairement? * Alcohol or drugs * Violations issued"

Are those are just suggestions? For mine I did, Collisionmanner with a combination of time and weather, do you want me to add light, surface and junction?

**Comment by jamisoncrawford** [comment]
Hi Jacob, yes - those are suggestions, but you can create new variables or integrate external data if you feel particularly motivated!  I'm going to post the grading rubric on Canvas now, but these are the grading criteria:


* **New tabs added to dashboard:** Custom tabs successfully integrated into the template, RMD file runs without errors (10 pts) 
* **Widget integration:** Widgets correctly linked to data filters and output presented on pages (8 pts)
* **Data reporting:** Value boxes, descriptive tables, custom graphics, or other data reporting is provided and functional(4 pts) 
* **Documentation:** Sufficient documentation provided on about tab (2 pts) 
* **Style points:** Dashboard content exceeds expectations in functionality, design, page layouts, analysis, or insights provided (3 pts)
* **Upload to shinyapps.io:** Dashboard posted to Shinyapps.io and available through an active URL (3 pts)

Total: 30 pts

Let me know if I can clarify on any of these - grading is relatively lenient so don't feel you have to go way above and beyond the call of duty, but do treat this as a time to experiment with `flexdashboard` and Shiny!

**Comment by afullerton** [comment]
Thanks for sharing - I have a quick clarifying question. Are we adding two or three new tabs? The instructions say that there are four tabs given to us and that we need to create six in total. I see three tabs given to us with data so if we added two we'd be at 5 tabs. I wasn't sure if the about tab counted towards the total of six or not. 

Thanks!

**Comment by jamisoncrawford** [comment]
@afullerton good catch!  You only have to add two new tabs.  The _reformatted (new) template_ only has three tabs with data visualizations:

* Traffic Accidents By Dat and Time
* Driver Characteristics
* Comparisons

In addition, there are two metadata and documentation tabs:

* About
* Data 

So the template already has five (5) tabs.  The project is to add two (2) more tabs, making a **total of seven (7) tabs**, five (5) of which are data visualizations/dashboard elements and two (2) of which are metadata/documentation.

Apologies for the emphasis - just want to be absolutely clear!

**Comment by afullerton** [comment]
Super helpful, thank you @jamisoncrawford
